components/map.jsx

```diff
@@ -110,6 +110,7 @@ var MarkerPopupClub = React.createClass({
         <br/>
         <br/>
         <p>{this.props.description}</p>
+        <p><small>Led by <a href={"https://webmaker.org/en-US/search?type=user&q=" + this.props.owner}>{this.props.owner}</a></small></p>
```

Comment:
Hmm, it would probably be good to add a helper function (or maybe even a super simple react component) somewhere that returns the profile URL for a user, to avoid DRY breakage between this link and the one in the clubs list, but I don't think it's a blocker or anything. R+ I say, R+!